compute/admin_guide/configure/agentless-permission-templates.adoc

```diff
@@ -0,0 +1,68 @@
+== Agentless Permission Templates
+
+Agentless scanning requires specific permissions to work on your self-hosted deployments.
+Different platforms have different permission requirements. 
+To make applying the right permissions easier, you can download permission templates for your specific platforms.
+The following sections explain the different permission templates and what they entail.
+
+=== AWS
+
+You need to download three permission templates as JSON files for AWS cloud account scanning. The templates are needed to enable the different ways accounts are scanned: independent scanning, scan hub account, and target account.
+
+==== Independent Scanning
+
+All scanning happens within the same account.
+This account is independent of other accounts.
+The account requires all  needed permissions to create snapshots, create scanners, etc. 
+The permission template you need to download ends in `_target_user_permissions.json`.
+
```

Comment:
Can we add a heading equivalent to independent scanning and call it "Scan Hub setup"

Reply:
I've added the subheadings and context.